webpay/auth/decorators.py

```diff
@@ -132,11 +132,6 @@ def get_locked_step(request, step):
     :rtype: HttpResponse or None or False
 
     """
-    try:
-        step_index = flow['locked'].index(step)
-    except ValueError:
-        step_index = -1
-
```

Comment:
All this will hopefully get a proper cleanup in https://bugzilla.mozilla.org/show_bug.cgi?id=1130465